KS-4-2 Checking-is-cargo-average

CasTeTRU opened the pull request:
Checking is cargo weight is between 35kg and 100kg and volume is less than 100l

CasTeTRU committed:
KS-4-2 Checking is cargo average
Checking is cargo weight is between 35kg and 100kg and volume is less than 100l

ievetrovdev commented on src/main/kotlin/lesson_4/lesson4_task2.kt:
```diff
@@ -0,0 +1,13 @@
+    var cargoWeight = 20
+    var cargoVolume = 80
+    println("Груз с весом $cargoWeight кг и объемом $cargoVolume л соответствует категории 'Average':" +
+            " ${35 < cargoWeight && cargoWeight <= 100 && cargoVolume < 100}")
```
Значения в этом коде являются [магическими числами](https://ru.wikipedia.org/wiki/%D0%9C%D0%B0%D0%B3%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%BE%D0%B5_%D1%87%D0%B8%D1%81%D0%BB%D0%BE_(%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5)#%D0%9F%D0%BB%D0%BE%D1%85%D0%B0%D1%8F_%D0%BF%D1%80%D0%B0%D0%BA%D1%82%D0%B8%D0%BA%D0%B0_%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F)
не лучшая практика. добавь, пожалуйста, переменные)

CasTeTRU committed:
KS-4-2 Added new variables instead of magic numbers
Значениям минимального и максимального веса и обьема присвоены переменные. Убраны магические числа
